sql/plan/ddl.go

```diff
@@ -651,7 +651,7 @@ func (d *DropTable) WithTriggers(triggers []string) sql.Node {
 func (d *DropTable) TableNames() ([]string, error) {
 	tblNames := make([]string, len(d.Tables))
 	for i, t := range d.Tables {
-		// either *ResolvedTable OR *UnresolvedTable here
+		// either *TableNode OR *UnresolvedTable here
```

Comment:
as written, i think UnresolvedTable is a TableNode? Also nit TableNode is an interface, no *

Reply:
Changed to be ResolvedTable instead of TableNode, since we only want to drop actual tables, not table functions.